### PR TITLE
[mobile] Fix the checkbox dialog gap according to the GUI guide

### PR DESCRIPTION
**Pull request by DohyungLim**
- List only applies under the ui-popup-body-checkbox class.

### CODE REVIEW

**Comment by DohyungLim** [comment]
Snapshot : 

image

File: src/css/profile/mobile/common/popup.less [hunk 1]
```diff
@@ -109,7 +109,25 @@
 		}
 
 		.ui-popup-body-checkbox {
-			margin-top: 8px;
+			margin-top: 8 * @px_base;
+
+			& .ui-listview.ui-content-area {
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: Is the '&' neccessary?

File: src/css/profile/mobile/common/popup.less [hunk 2]
```diff
@@ -109,7 +109,25 @@
...
+				height: 32 * @px_base;
+
+				& .ui-li-has-checkbox {
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: ditto.

File: src/css/profile/mobile/common/popup.less [hunk 3]
```diff
@@ -109,7 +109,25 @@
...
+					height: 32 * @px_base;
+
+					& .ui-li-text {
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: ditto.

File: src/css/profile/mobile/common/popup.less [hunk 4]
```diff
@@ -109,7 +109,25 @@
...
+						margin: 5 * @px_base 0 0 0;
+						padding: 0;
+
+						& .ui-li-text-title {
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less]: ditto.

File: src/css/profile/mobile/common/popup.less (hunk 1 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: no. not necessary.
I removed.

done.

File: src/css/profile/mobile/common/popup.less (hunk 2 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: done.

File: src/css/profile/mobile/common/popup.less (hunk 3 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: done.

File: src/css/profile/mobile/common/popup.less (hunk 4 above)
Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less]: done.

File: src/css/profile/mobile/common/popup.less [hunk 5]
```diff
@@ -109,7 +109,25 @@
...
+						margin: 5 * @px_base 0 0 0;
+						padding: 0;
+
+						.ui-li-text-title {
+							height: 23 * @px_base;
+							line-height: 23 * @px_base;
+						}
+					}
+				}
+			}
```
Comment (HunseopJeong) [inline src/css/profile/mobile/common/popup.less:122]: I have a question about dialog.html.
Why do you use the list element only for the checkbox?
``` html
				<div class="ui-popup-content">
					Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt
					<div class="ui-popup-body-text ui-popup-body-checkbox">
						<ul class="ui-listview ui-content-area">
							<li class="ui-li-has-checkbox">
								<input checked="checked" class="checkboxes" name="checkbox-1" type="checkbox" />
								<div class="ui-li-text">
									<span class="ui-li-text-title" id="checkbox-text-1">
										Do not show again
									</span>
								</div>
							</li>
						</ul>
					</div>
```

Comment (DohyungLim) [reply src/css/profile/mobile/common/popup.less:130]: @HunseopJeong 
Does the checkbox you commented mean "ui-popup-body-checkbox"?

**Review by HunseopJeong — APPROVED** [review]
It looks much better than before!

**Pull request merged**